RUFAS/data_validator.py

```diff
@@ -1626,6 +1626,16 @@ class CrossValidator:
     def __init__(self) -> None:
         self._alias_pool: dict[str, Any] = {}
         self._event_logs: list[dict[str, str | dict[str, str]]] = []
+        self.relation_mapping: dict[str, Any] = {
```

Comment:
Couple things here:

1. Can you remind me why you went with the attribute mapping vs. global mapping?
2. Can you add a docstrings attributes section for this class now that we know the attributes?